lib/workers/pr/index.ts

```diff
@@ -149,6 +149,16 @@ export type EnsurePrResult = ResultWithPr | ResultWithoutPr;
 export async function ensurePr(
```

Comment:
Maybe this PR fixes issue https://github.com/renovatebot/renovate/issues/12561 as well?

Reply:
Added to the list, thanks